libs/event/src/lib/form/shell/shell.component.ts

```diff
@@ -164,26 +183,31 @@ export class EventFormShellComponent implements OnInit, OnDestroy {
     return outlet?.activatedRouteData?.animation;
   }
 
+  private setMissing(missing?: Part) {
+    for (const part of parts) {
+      this.missing[part] = missing === part;
+    }
+  }
+
   async checkTitleAndScreener(titleId: string) {
     if (!titleId) {
-      this.errorChipMessage = 'No title selected';
-    } else {
-      const title = await this.movieService.getValue(titleId);
-      if (!title.promotional.videos?.screener?.jwPlayerId) {
-        this.errorChipMessage = 'Screening file missing';
-      } else if (title.app.festival.status === 'draft') {
-        // Titles in draft are not allowed for screenings
-        this.errorChipMessage = 'No title selected';
-      } else {
-        this.errorChipMessage = '';
-      }
+      return this.setMissing('title');
+    }
+
+    const title = await this.movieService.getValue(titleId);
+    if (!title.promotional.videos?.screener?.jwPlayerId) {
+      return this.setMissing('screener');
     }
```

Comment:
```suggestion
    if (!title.promotional.videos?.screener?.jwPlayerId)  return this.setMissing('screener');
```


or, for all cases
```
let missing = '':

 if (!title.promotional.videos?.screener?.jwPlayerId) missing = 'screener';
...
if (title.app.festival.status === 'draft')  missing = 'title';

```

and at the end 
`return this.setMissing('missing');`

as you want ;)